UserManual.md

```diff
@@ -0,0 +1,877 @@
+  Color and black/white images are supported
+  
+  8 bit depth
+ 
+### Output Options:
+Image sequencer supports the following options as output images:
+  * PNG
+  * GIF (provides images thru each sequence)   
+
+ 
+### Color Scheme – RGBA:
+Color is represented as as ‘tuple’ in (R,G,B,A) format. Each color channel (R-Red, B-Blue, G-Green)  has a value between 0 and 255.
+*‘A’ represents transparency with values 0 to 255.*
+
+For example: 
+black is: (0,0,0),
+white is (255,255,255),
+red is (255,0,0), 
+blue is (0,0,255). 
+
+### Image coordinates:
+Coordinates in (X,Y), which is (0,0) is located at the top left of the image. 
+
+
+![graph](https://lh3.googleusercontent.com/_86j-laf9fp4rVNTVKmdOprVAeAeAVzRryWJO2R3X3B7apoKg409NKx1VwDfl9Y5tWNH0IcqzJKR "grpah")
+
+
+Figure 1 Public Lab Image Sequencer coordinate matrix
+
+## Processing Environment:
+Image Sequencer modules are designed to be run either in the browser or in a Node.js environment. 
+
+*(What is the processing environment ???)*
+*(processing time vs file size)*
+
+## Contributions:
+Image Sequencer is an open source tool and is happily accepting contributions for improvement.   
+* See https://github.com/publiclab/image-sequencer/blob/main/CONTRIBUTING.md for more information. 
+* If you find a bug, please list it at https://github.com/publiclab/image-sequencer/issues/new, and help  develop Image Sequencer by opening an issue!
+
+## Image Modules:
+
+Image Sequencer uses modules that perform a variety of step-by-step image processing tasks.  Figure 2 describes the general workflow.
+
+This section describes each module and a general implementation example.  Most examples used Figure 3 as the input reference image.  Advanced image processing concepts include a wikipedia reference for mor information. 
+
+![workflow](https://lh3.googleusercontent.com/XE_twkHy8uxZ6jycIddQlMxJF91AbU29pJPILcRcolwiiaNJAUPrJhU_PAJYmVGGoHcJ7aOSTNFR)
+
+
+Figure 2 Image Sequencer workflow:
+
+
+![Figure 4 Reference Image](https://lh3.googleusercontent.com/A98oLhF19A1li9ugUa9lvyOJXzKang6nwAn61E_Zwf6NZ3Tqw3OsuuHqUbbb2MnWD_eTL140rRJj )
+
+Figure 3 Image Sequencer general reference image used as example input.
+
+Additional information on the modules and other operating modes can be found at the Image Sequencer GitHub site. 
+
+ - [https://github.com/publiclab/image-sequencer/tree/main/docs](https://github.com/publiclab/image-sequencer/tree/main/docs)
+ - [https://github.com/publiclab/image-sequencer/blob/main/README.md](https://github.com/publiclab/image-sequencer/blob/main/README.md)
+### Add-qr-module:
+This module adds a QR code to the image that corresponding to an input string.
+
+Where options is an object with the following properties:
+ - size: size of QR code in pixels (default 200)
+ - qrCodeString: input string to generate QR code
+ 
+ 
+![qr](https://lh3.googleusercontent.com/o99UIUmc8jbs7sn7InSG0NfizUbm3UhOe5q96dPPEGo4ptqHs0lULdkEoFdEEMOLC17dqxFn89wy)
+
+
+Usage:
+
+    sequencer.loadImage('PATH')
```

Comment:
One nice thing you can do is add three of the \` character around sections that are code examples, and even **\`\`\`js** above and **\`\`\`** below, which makes it syntax highlighted for javascript! See the existing README on image sequencer for examples!

Reply:
```js
// example code:
do();
```